packages/api-bindings/src/lens/utils/media.ts

```diff
@@ -0,0 +1,24 @@
+import { Brand } from '@lens-protocol/shared-kernel';
+
+export type ImageSizeTransform = Brand<string, 'ImageSizeTransform'>;
```

Comment:
We could type this so that informs TS consumers of the valid options: 

```ts
type ImageSizeTransform = `${number}px` | `${number}%` | 'auto'
```

This is not super robust as `number` could be hex numbers, decimals, `1_000` notations, and many others.

A more robust and strict could be:

```ts
type Digit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'

type Percentage = `100%` | `${Digit}${Digit}%` | `${Digit}%`

type Pixel = `${Digit}${Digit}${Digit}${Digit}px` | `${Digit}${Digit}${Digit}px` | `${Digit}${Digit}px`

type ImageSizeTransform = Pixel | Percentage | 'auto'
```

Reply:
I am making assumption that pixel values in the single digit or above 4 digits are not meaningful... probably safe for any reasonable application.